code/modules/vehicles/pimpin_ride.dm

```diff
@@ -76,12 +76,67 @@
 	if(floorbuffer)
 		add_overlay("cart_buffer")
 
-
 /obj/vehicle/janicart/attack_hand(mob/user)
 	if(..())
 		return 1
 	else if(mybag)
 		mybag.loc = get_turf(user)
 		user.put_in_hands(mybag)
 		mybag = null
-		update_icon()
\ No newline at end of file
+		update_icon()
+
+/obj/vehicle/lawnmower
+	name = "lawn mower"
+	desc = "Equipped with realible safeties to prevent <i>accidents</i> in the workplace."
+	icon = 'icons/hippie/obj/vehicles.dmi'
+	icon_state = "lawnmower"
+	var/emagged = FALSE
+	var/list/drive_sounds = list('sound/effects/mowermove1.ogg', 'sound/effects/mowermove2.ogg')
+	var/list/gib_sounds = list('sound/effects/mowermovesquish.ogg')
+	var/driver
+
+/obj/vehicle/lawnmower/emagged
+	emagged = TRUE
+
+/obj/vehicle/lawnmower/emag_act(mob/user)
+	if(emagged)
+		user << "<span class='warning'>The safety mechanisms on \the [src] are already disabled!</span>"
+		return
+	user << "<span class='warning'>You disable the safety mechanisms on \the [src].</span>"
+	emagged = TRUE
+
+/obj/vehicle/lawnmower/buckle_mob(mob/living/buckled_mob, force = 0, check_loc = 0)
+	. = ..()
+	riding_datum = new/datum/riding/lawnmower
+
+/obj/vehicle/lawnmower/Bump(atom/A)
+	if(emagged)
+		if(isliving(A))
+			var/mob/living/M = A
+			M.adjustBruteLoss(25)
+			var/atom/newLoc = get_edge_target_turf(M, get_dir(src, get_step_away(M, src)))
+			M.throw_at(newLoc, 4, 1)
+
+/obj/vehicle/lawnmower/Move()
+	..()
+	var/gibbed = FALSE
+	var/mob/living/carbon/H
+
+	if(has_buckled_mobs())
+		H = buckled_mobs[1]
+
+	if(emagged)
+		for(var/mob/living/carbon/human/M in loc)
```

Comment:
`bump()` is not being called if you drive over someone, prove it does and I'll happily change it

_also_ [nice stolen avatar](https://forum.ss13.co/showthread.php?tid=4964&pid=73147#pid73147)